api-lambda-ses/src/index.py

```diff
@@ -0,0 +1,89 @@
+import boto3
+import os
+from botocore.exceptions import ClientError
+    
+# This address must be verified with Amazon SES.
+SENDER = os.environ['SENDER']
+
+# is still in the sandbox, this address must be verified.
+RECIPIENT = os.environ['RECIPIENT']
+
+# Specify a configuration set. If you do not want to use a configuration
+# set, comment the following variable, and the 
+# ConfigurationSetName=CONFIGURATION_SET argument below.
+#CONFIGURATION_SET = "ConfigSet"
+
+# If necessary, replace us-west-2 with the AWS Region you're using for Amazon SES.
+AWS_REGION = "us-east-1"
+
+# The subject line for the email.
+SUBJECT = "Amazon SES Test (SDK for Python)"
+
+# The email body for recipients with non-HTML email clients.
+BODY_TEXT = ("Amazon SES Test (Python)\r\n"
+             "This email was sent with Amazon SES using the "
+             "AWS SDK for Python (Boto)."
+            )
+            
+# The HTML body of the email.
+BODY_HTML = """<html>
+<head></head>
+<body>
+  <h1>Amazon SES Test (SDK for Python)</h1>
+  <p>This email was sent with
+    <a href='https://aws.amazon.com/ses/'>Amazon SES</a> using the
+    <a href='https://aws.amazon.com/sdk-for-python/'>
+      AWS SDK for Python (Boto)</a>.</p>
+</body>
+</html>
+            """            
+
+# The character encoding for the email.
+CHARSET = "UTF-8"
+
+# Create a new SES resource and specify a region.
+client = boto3.client('ses',region_name=AWS_REGION)
```

Comment:
I don't think you need to specify the region if you are connecting to the region you are running from.

Reply:
We sometime might face errors while deploying stack in different region and if our verified email is in a different region. Its better to mention the region.